### PR TITLE
BGA-317 - Added random default node behavior for hbar tx

### PR DESCRIPTION
**Pull request by gonzaalvarez70**
Ticket: BGA-317

### CODE REVIEW

File: modules/account-lib/src/coin/hbar/utils.ts [hunk 1]
```diff
@@ -2,10 +2,41 @@ import * as _ from 'lodash';
 import { Ed25519PublicKey, TransactionId, AccountId } from '@hashgraph/sdk';
 import BigNumber from 'bignumber.js';
 import * as hex from '@stablelib/hex';
+import { NetworkType } from '@bitgo/statics';
 import { AccountID, Timestamp } from './ifaces';
 
 const MAX_TINYBARS_AMOUNT = new BigNumber(2).pow(63).minus(1);
 
+const mainnetNodes: string[] = [
```
Comment (twtaylorbitgo) [inline modules/account-lib/src/coin/hbar/utils.ts:10]: can you reference where we pull these from?

File: modules/account-lib/src/coin/hbar/utils.ts [hunk 2]
```diff
@@ -2,10 +2,41 @@ import * as _ from 'lodash';
...
+  '0.0.6',
+  '0.0.7',
+  '0.0.8',
+  '0.0.9',
+  '0.0.10',
+  '0.0.11',
+  '0.0.12',
+];
+
+const testnetNodes: string[] = ['0.0.3', '0.0.4', '0.0.5', '0.0.6'];
```
Comment (twtaylorbitgo) [inline modules/account-lib/src/coin/hbar/utils.ts:23]: same here

File: modules/account-lib/src/coin/hbar/utils.ts (hunk 1 above)
Comment (gonzaalvarez70) [reply modules/account-lib/src/coin/hbar/utils.ts:10]: Yeah sure those are extracted from the hedera SDK index-node client. Maybe I can add a comment explicitly saying where we got them from.

**Comment by tylerlevine** [comment]
@gonzaalvarez70 is this PR still active or can it be closed?

**Comment by tylerlevine** [comment]
Closing, as this is stale with unresolved merge conflicts.

**Pull request closed without merging**